src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -817,4 +817,10 @@
   <data name="FancyZones_WindowBehavior_GroupSettings.Text" xml:space="preserve">
     <value>Window behavior</value>
   </data>
+  <data name="PowerRename_BehaviorHeader.Text" xml:space="preserve">
+    <value>Behavior</value>
+  </data>
+  <data name="PowerRename_Toggle_UseBoostLib.Content" xml:space="preserve">
+    <value>Use Boost Library (provides extended features but may use different regex syntax)</value>
+  </data>
```

Comment:
Let's change `Library` to lowercase, and also add a comment to help the translation team:
```
  <data name="PowerRename_Toggle_UseBoostLib.Content" xml:space="preserve">
    <value>Use Boost library (provides extended features but may use different regex syntax)</value>
     <comment>Boost is a product name, should not be translated</comment>
  </data>
```

Reply:
This hasn't been fixed yet (There were two places where we have same strings).

Reply:
Updated this too.